src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -94,23 +95,23 @@ namespace winrt::TerminalApp::implementation
         void _HookupKeyBindings(TerminalApp::AppKeyBindings bindings) noexcept;
         void _RegisterActionCallbacks();
 
-        void _UpdateTitle(std::shared_ptr<Tab> tab);
-        void _UpdateTabIcon(std::shared_ptr<Tab> tab);
+        void _UpdateTitle(const Tab& tab);
```

Comment:
why's this one const but the others aren't?

Reply:
`_UpdateTitle` doesn't change the tab directly, it actually just calls the handlers. `UpdateTabIcon` directly calls the tab's `UpdateIcon` function, so it couldn't be const 😢